### ISSUE TITLE
if meeting point is over large body of water, land units stay at home

### ISSUE THREAD

**Issue by andy5995** [MegaGlest/megaglest-source#164]
I realize this is very trivial but wanted to make a note of it.

If a meeting point is over water, land units stay at home.

Example where this could be a problem:

Obelisk meeting point is over water mid-way through the map.

Ibis go there, but snakes and scarabs don't move anywhere if selected for production. I thought the desired behavior would be for the land units to move as far as they could - near the water - and then stop.

Related to this are the comments I left on https://github.com/MegaGlest/megaglest-source/commit/438d37c59f7d22d70bceb39e35f20b5efcbec851 So trivial I am reluctant to create a separate ticket for it.

**Comment by titiger** [comment]
I was not able to reproduce the "units don't move" problem. For me the units always moved. Maybe you had a special case were meeting point was so far away and so unreachable that the pathfinder gave up.
I see no need for any action on this atm .

**Comment by andy5995** [comment]
Right, definitely does not need to be considered right now, esp. before the Steam release. I had just intended to make a note for the future.

Yes, the target is an unreachable point. When this is possibly a problem is if a user is making Ibis, telling the Ibis to meet over a point unreachable for land units (such as a large body of water), but then if the user switches to producing scarabs or snakes. The user is likely to expect those land units to at least come *near* the body of water, but instead they would just stay at home.

So clearly not a priority right now. If after this additional summary I wrote that you think no change needs to be made, then this ticket can be closed.
